### PR TITLE
[ready-to-merge] New Siem Overview

### PR DESCRIPTION
**Pull request by edwardkoval**

### CODE REVIEW

File: concepts/security-concept-overview.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The [Microsoft Graph Security API](/graph/api/resources/security-api-overview?vi
...
+Connect once to integrate alerts from any Microsoft Graph-integrated security solution and keep alert status and assignments in sync across all solutions. You can also stream alerts to security information and event management (SIEM) solutions, such as Splunk and IBM QRadar, via [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitor-stream-monitoring-data-event-hubs#what-can-i-do-with-the-monitoring-data-being-sent-to-my-event-hub). For more info about solution integrations with the security API entities, see [Security solution integrations using the Microsoft Graph Security API](security-integration.md).
```
Comment (preetikr) [inline concepts/security-concept-overview.md]: @CelesteDG - One quick update as I read through this - Please remove the text 
'and IBM QRadar, via [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitor-stream-monitoring-data-event-hubs#what-can-i-do-with-the-monitoring-data-being-sent-to-my-event-hub)' 
and replace with
'using Microsoft Graph Security API connectors'
Thanks!

Comment (CelesteDG) [reply concepts/security-concept-overview.md]: Hi @preetikr - How about Splunk? I saw that that topic was also removed. Should I remove that part as well and just have "such as using Microsoft Graph Security API connectors" ?

Comment (CelesteDG) [reply concepts/security-concept-overview.md]: Hmm. I see an entry for SIEM - Splunk in the list of connectors table so please ignore that question.

**Comment by CelesteDG** [comment]
@preetikr and @edwardkoval - Please see the last build, click the Preview URL, for a final sanity check. All 3 deleted files should be redirecting to security-integration.md.

File: concepts/security-concept-overview.md [hunk 2]
```diff
@@ -30,7 +28,7 @@ The [Microsoft Graph Security API](/graph/api/resources/security-api-overview?vi
 
 ### Unify and standardize alert tracking
 
-Connect once to integrate alerts from any Microsoft Graph-integrated security solution and keep alert status and assignments in sync across all solutions. You can also stream alerts to security information and event management (SIEM) solutions, such as Splunk and IBM QRadar, via [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitor-stream-monitoring-data-event-hubs#what-can-i-do-with-the-monitoring-data-being-sent-to-my-event-hub). For more info about solution integrations with the security API entities, see [Security solution integrations using the Microsoft Graph Security API](security-integration.md).
+Connect once to integrate alerts from any Microsoft Graph-integrated security solution and keep alert status and assignments in sync across all solutions. You can also stream alerts to security information and event management (SIEM) solutions, such as Splunk and using Microsoft Graph Security API connectors. For more info about solution integrations with the security API entities, see [Security solution integrations using the Microsoft Graph Security API](security-integration.md).
```
Comment (preetikr) [inline concepts/security-concept-overview.md]: Minor tweak - "and" needs to be removed from "such as Splunk and using Microsoft Graph..."
It needs ot be "such as Splunk using Microsoft Graph..."

File: .openpublishing.redirection.json [hunk 3]
```diff
@@ -26745,6 +26745,11 @@
             "redirect_url": "/graph/security-integration",
             "redirect_document_id": false
         },
+        {
```
Comment (preetikr) [inline .openpublishing.redirection.json]: Want to ensure the redirection resolves the following 3 links to point to graph/security-integration.md. 
1. https://docs.microsoft.com/en-us/graph/security-siemintegration 
2. https://docs.microsoft.com/en-us/graph/security-splunk-siemintegration  
3. https://docs.microsoft.com/en-us/graph/security-qradar-siemintegration

Comment (CelesteDG) [reply .openpublishing.redirection.json]: @preetikr - They do. If you click the Preview URL link for these, they show the security-integration topic.

**Pull request merged**